R/pkg/R/DataFrame.R

```diff
@@ -1146,7 +1147,7 @@ setMethod("head",
 
 #' Return the first row of a SparkDataFrame
 #'
-#' @param x A SparkDataFrame
```

Comment:
I don't have a good answer yet. When I tried to move the param there, it seems that the generic functions for RDD Actions and Transformations are not exposed. Do you know specific reason for that by chance?